sirepo/pkcli/job_agent.py

```diff
@@ -751,6 +751,19 @@ def _processor():
 if [[ ! $LD_LIBRARY_PATH =~ /usr/lib64/mpich/lib ]]; then
     export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/lib64/mpich/lib
 fi
+export LD_LIBRARY_PATH=/opt/cray/pe/mpt/7.7.19/gni/mpich-gnu-abi/8.2/lib:$LD_LIBRARY_PATH
```

Comment:
This one should be documented too, e.g. libmpi_..?

Make a for loop? Perhaps it's time to add something like rs_shifter to bashrc?

```sh
declare -a x=(
    # libxpmem.so.0
    xpmem/2.2.27-7.0.3.1_3.4__gada73ac.ari
...
)
for y in "${x[@]}"; do
    LD_LIBRARY_PATH=/opt/cray/$y/lib64:$LD_LIBRARY_PATH
done
export LD_LIBRARY_PATH
```

Reply:
Turns out that one (imports libmpi) is the only one that is needed https://github.com/radiasoft/sirepo/pull/4357/commits/87ef691b2e109cfb81600600cc11a39568aab666

Reply:
I think this should be like it was, that is, it shouldn't reinsert in the path if already there.

Reply:
https://github.com/radiasoft/sirepo/pull/4357/commits/78958ee6dcf1e1756d101755c8437c86914edcea